guides/micronaut-scope-types/micronaut-scope-types.adoc

```diff
@@ -0,0 +1,139 @@
+
+== Introduction
+
+Micronaut features an extensible bean scoping mechanism based on https://javax-inject.github.io/javax-inject/[JSR-330]. 
+
+
+== Scenario
+
+We use the following scenario to talk about the different types of scopes.
+
+The following `@Controller` injects two collaborators. 
+
+source:singleton/RobotController[tags=pkg|imports|controller|clazz]
+
+Each collaborator has an injection endpoint for a bean of type `Robot`.
+
+source:singleton/RobotFather[]
+
+source:singleton/RobotMother[]
+
+Let's discuss how the application behaves depending on the scope used for the bean of type `Robot`.
+
+== Singleton
+
+**Singleton scope indicates only one instance of the bean will exist**. 
+
+To define a singleton, annotate a class with `jakarta.inject.Singleton` at the class level. 
+
+The following class creates a unique identifier in the constructor. This identifier allows us to identify how many   `Robot` instances are used. 
+
+source:singleton/Robot[tags=pkg|clazz]
+callout:singleton[1]
+
+=== Singleton Test
+
+The following test verifies `@Singleton` behavior.
+  
+test:SingletonScopeTest[tags=imports|testheaders|test]
+callout:micronaut-test[1]
+callout:http-client[2]
+<3> The same instance fulfills both injection points at `RobotFather` and `RobotMother`. 
+<4> Same instance is used upon subsequent requests.
+
+== Prototype
+
+**Prototype scope indicates that a new instance of the bean is created each time it is injected**
+
+Let's use `@Prototype` instead of `@Singleton`. 
+
+source:prototype/Robot[]
+callout:prototype[1]
+
+=== Prototype Tests
+
+The following test verifies the behavior of `Prototype` scope. 
+
+test:PrototypeScopeTest[tags=imports|testheader|test]
+callout:micronaut-test[1]
+callout:http-client[2]
+<3> A new instance is created to fulfill each injection point. Two instances - one for `RobotFather` and another for `RobotMother`. 
+<4> Instances remain upon subsequent requests.
+
+== Request
+
+**@RequestScope scope is a custom scope that indicates a new instance of the bean is created and associated with each HTTP request**
+
+source:request/Robot[]
+callout:request-scope[1]
+callout:request-aware[2]
+
+=== Request Tests
+
+The following test verifies the behavior of `@RequestScope` scope. 
+
+test:RequestScopeTest[tags=imports|testheader|test]
+callout:micronaut-test[1]
+<3> Both injection points, `RobotFather` and `RobotMother`, are fulfilled with the same instance of `Robot`. An instance associated with the HTTP Request. 
+<4> Both injection points are fulfilled with the new instance of `Robot`.
+
+
+== Refreshable
+
+**https://docs.micronaut.io/latest/guide/#refreshable[Refreshable scope] is a custom scope that allows a bean's state to be refreshed via the https://docs.micronaut.io/latest/guide/#refreshEndpoint[/refresh endpoint].**
+
+source:refreshable/Robot[]
+
+Your application needs the `management` dependency to enable the refresh endpoint.
```

Comment:
I would also mention that it is sensitive by default

Reply:
done https://github.com/micronaut-projects/micronaut-guides/pull/1085/commits/695b51cc0c703d873f8d0a660d1b351fb3465334